Projects/Foundations/Schemas/Docs-Tutorials/I/Intro/Intro-to-Trading-Systems/intro-to-trading-systems-001-intro-to-trading-systems-tutorial.json

```diff
@@ -17,9 +17,9 @@
                 "updated": 1639147057219
             },
             {
-                "language": "TR",
-                "text": "Bu öğretici, size ticaret sistemlerinin nasıl oluşturulacağını öğreten bir dizi öğreticinin ilkidir. Seri, birden fazla konuyu derinlemesine ele alır ve temel ticaret bilgilerini belirli Superalgos uygulamalarıyla eşleştirmeyi amaçlayan bir ilerlemeye götürür. Bu ilk eğitim teorik kavramları kapsar, Superalgos'ta alım satım sistemlerinin nasıl yapılandırıldığını açıklar ve sistemde yerleşik alım satım çerçevesinden en iyi şekilde yararlanmak için stratejilerinizi nasıl çerçevelemeniz gerektiğini vurgular.",
-                "updated": 1640690079795
+                "language": "RU",
+                "text": "Этот учебник является первым в серии учебников, которые научат вас создавать торговые системы. Серия углубляется во множество тем и проводит вас через последовательность, которая призвана соотнести базовые знания о торговле с конкретными реализациями Superalgos. В этом первом уроке рассматриваются теоретические концепции, объясняется, как построены торговые системы в Superalgos, и подчеркивается, как нужно строить свои стратегии, чтобы извлечь максимальную пользу из торговой платформы, встроенной в систему.",
+                "updated": 1640878603688
```

Comment:
It seems like you are accidentally overwriting the Turkish Translation. Do you know how to fix this overwriting problem? Otherwise let us know so we can help you.